packages/pages/src/upgrade/upgrade.ts

```diff
@@ -12,8 +25,20 @@ interface UpgradeArgs {
 
 const handler = async (args: UpgradeArgs) => {
   const scoped = { scope: args.scope || "" };
+  const source = path.resolve(scoped.scope);
   const projectStructure = await ProjectStructure.init(scoped);
-  await updatePages(projectStructure);
+  await updateSitesComponents(source);
+  await updateDevDependencies(source);
+  checkLegacyMarkdown(source);
+  replacePagesSlashComponentsImport(source);
+  replaceSitesComponentsImports(source);
+  removeFetchImport(source);
+  updatePackageScripts(source);
+  updatePackageEngines(source);
+  checkNodeVersion();
+  await updatePages(source);
```

Comment:
Give this a more descriptive name.

Reply:
Would it be updateYextPages? This is the command that updates @yext/pages to the version that is called by npx pages upgrade

Reply:
updatePagesJSToCurrentVersion